test/example.css

```diff
@@ -2,6 +2,10 @@ html {
     box-sizing: border-box;
 }
 
+a {
+    animation: 100ms;
```

Comment:
what is this block demonstrating? one property doesn't feel like enough since so much of this config is about ordering.